### DISCUSSION TITLE
New translation in Tagalog and Iloko

### DISCUSSION THREAD

**Discussion by ghost** [sircharlo/meeting-media-manager#2271] [Translations]
### What language do you speak?

I would like to help translate M³ into Tagalog (tl) and Iloko (ilo)

**Comment by mtdvlpr** [comment]
Hi @bridenkenn, thanks for your willing spirit! Please [open Crowdin](https://crowdin.com/project/meeting-media-manager), log in with GitHub and request to join our translation project. I've prepared your languages for you.

**Comment by mtdvlpr** [comment]
Hi @bridenkenn, you can test your Tagalog translations by installing [the latest beta release](https://github.com/sircharlo/meeting-media-manager/releases)

**Comment by mtdvlpr** [comment]
Hi @bridenkenn, I'm about to officially release your translations for Tagalog and Iloko. Thank you for your work! For date functions, we rely on Dayjs which has its own translations for date related words, like week days. Dayjs doesn't support Iloko (yet). If you want the days of the week to also be translated in M3, you could [contribute to Dayjs](https://github.com/iamkun/dayjs/tree/dev/src/locale).
